structopt-derive/src/lib.rs

```diff
@@ -278,17 +277,23 @@
 //! }
 //! ```
 //!
-//! There are four kinds custom string parsers:
+//! There are five kinds of custom parsers:
 //!
 //! | Kind              | Signature                             | Default                         |
 //! |-------------------|---------------------------------------|---------------------------------|
 //! | `from_str`        | `fn(&str) -> T`                       | `::std::convert::From::from`    |
 //! | `try_from_str`    | `fn(&str) -> Result<T, E>`            | `::std::str::FromStr::from_str` |
 //! | `from_os_str`     | `fn(&OsStr) -> T`                     | `::std::convert::From::from`    |
 //! | `try_from_os_str` | `fn(&OsStr) -> Result<T, OsString>`   | (no default function)           |
+//! | `multiple`        | (no signature)                        | (no default function)           |
```

Comment:
| `from_occurrences` | `fn(u64) -> T` | `value as T` |

It would be great to have a consistent name as `from_occurrences` and have the possibility to give a conversion function (for transforming the number to an Enum for example).

Reply:
Sounds good! Will make this change (and the others) soon.

Reply:
We should probably take an fn(u64) -> Result<T, E> so that you can accept some number of flags and not other, i.e, if you only have three verbosity levels and want to enforce that.